parser/main.py

```diff
@@ -62,6 +61,10 @@
 INFLUX_DEBUG_BUCKET = ENV_CONFIG["INFLUX_DEBUG_BUCKET"]
 INFLUX_PROD_BUCKET = ENV_CONFIG["INFLUX_PROD_BUCKET"]
 
```

Comment:
All bucket names should be encoded in the .env file.

Reply:
Would this mean, for example, having buckets like `CAN_test` and `CAN_prod` keyed in the .env file? For example:

INFLUX_DEBUG_CAN_BUCKET="CAN_test"?

If so, I chose not to do that because we only need to access the debug endpoint of the parser and then the parser will decide on its own the correct bucket; we do not need to specify what buckets are available as the user. As bit of context, this is part of the previous implementation of sunlink which used the name of the bucket in the .env file to determine the  **exact** bucket to put the data in. However, in my implementation, I decided to abstract that ability by standardizing the debug buckets to <MESSAGE_NAME>_test and production buckets to <MESSAGE_NAME>_prod.

**Decision**: The cons to my implementation is that if somebody created a bucket named whatever they like (Ex. `my_bucket`) then they would have to go into `main.py` and change this line:

```python
write_api.write(bucket=message.type + bucket, org=INFLUX_ORG, record=point)
```
To be
```python
write_api.write(bucket="my_bucket", org=INFLUX_ORG, record=point)
```

This would be an inconvenience and has the potential for errors.

**Alternative Idea**: Make a single environment variable called `INFLUX_BUCKETS`. It will be formatted as "<BUCKET_NAME>, <BUCKET_NAME>, ... ". In doing this, I can have logic in the main.py of the parser to create an array/list out of these bucket names.
  * Benefit is that depending on the argument  (--debug, --prod for example) I can separate the bucket names by those that follow the naming convention of `_test` and `_prod` **AND** if there is a bucket name not following this convention (so my_bucket for example) I can make the parser **try** writing to this bucket (would only be successful if you made sure to manually create the bucket on InfluxDB since the create buckets script is not working).  
  * Essentially, all someone has to do in order for data to be written to their own bucket is 1) creating the bucket on influx and 2) adding the bucket name to their .env file as part of the `INFLUX_BUCKETS` string.
  
  What are your thoughts on this. Sorry for the long comment.